src/OmnichannelChatSDK.ts

```diff
@@ -1621,16 +1624,20 @@ class OmnichannelChatSDK {
         let conversationId;
 
         try {
-            const {LiveWSAndLiveChatEngJoin: liveWSAndLiveChatEngJoin} = this.liveChatConfig;
-            const {msdyn_postconversationsurveyenable, msfp_sourcesurveyidentifier, postConversationSurveyOwnerId} = liveWSAndLiveChatEngJoin;
+            const chatConfig: ChatConfig = this.liveChatConfig;
+            const {LiveWSAndLiveChatEngJoin: liveWSAndLiveChatEngJoin, ChatWidgetLanguage: chatWidgetLanguage} = chatConfig;
+            const {msdyn_postconversationsurveyenable, msfp_sourcesurveyidentifier, msfp_botsourcesurveyidentifier, postConversationSurveyOwnerId, postConversationBotSurveyOwnerId} = liveWSAndLiveChatEngJoin;
+            const {msdyn_localeid} = chatWidgetLanguage;
+            const localeId: string = msdyn_localeid ?? "1033";
 
-            if (msdyn_postconversationsurveyenable) {
+            if (msdyn_postconversationsurveyenable === "true") {
                 const liveWorkItemDetails = await this.OCClient.getLWIDetails(this.requestId);
-                const participantJoined: boolean = liveWorkItemDetails?.CanRenderPostChat && liveWorkItemDetails?.CanRenderPostChat === "True";
+                const participantJoined = liveWorkItemDetails?.CanRenderPostChat === "True";
+                const participantType = liveWorkItemDetails?.ParticipantType;
 
                 conversationId = liveWorkItemDetails?.ConversationId;
                 const surveyInviteLinkRequest = {
-                    "FormId": msfp_sourcesurveyidentifier,
+                    "FormId": participantType === "Bot" ? msfp_botsourcesurveyidentifier : msfp_sourcesurveyidentifier,
                     "ConversationId": conversationId,
                     "OCLocaleCode": getLocaleStringFromId(this.localeId) || getLocaleStringFromId(defaultLocaleId)
```

Comment:
Only `getLocaleStringFromId(this.localeId)` should be needed since `this.localeId` is set to `defaultLocaleId` by default.